Update docs for distributed tracing insights component (#15722)
newrelic/docs-website: src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

    alt="A screenshot depicting distributed tracing information on the APM summary page"
    src={apmDistributedTracingintheApmSummaryPage}
/>

The **Distributed tracing insights** chart shows you upstream and downstream entities that might cause increases in response time, error rate, or throughput of your service. 

> For example, let's say you want to investigate an increase in response time for a service, but the spike relates to the response time of an external call. If distributed tracing recorded a downstream entity that caused latency for your service around that time, you can view the entity and its change in performance in the list. Click the **View trace** button to see a distributed trace that shows performance changes. 

Reviewer: ```suggestion
For example, let's say you want to investigate an increase in response time for a service, but the spike relates to an external call. If distributed tracing recorded a downstream entity that caused latency for your service, you can view that change in performance in this list. Click the **View trace** button to see a distributed trace that shows performance changes. 
```

Author: change pushed up
